2020/05/index.js

```diff
@@ -1,3 +1,60 @@
+const { promises: fs } = require('fs');
+
+function convertToBinary(bsp) {
+  const binary = bsp
+    .split('')
+    .map(i =>
+      i === 'F' || i === 'L'
+        ? 0
+        : 1
+    )
+    .join('');
+
+  return parseInt(binary, 2);
+}
```

Comment:
@faizanminhas - we differed significantly in our solution here.

Reply:
Wow this is such a cool idea, nicely done